t/server/unifycr_meta_get_test.c

```diff
@@ -0,0 +1,61 @@
+#include <sys/types.h>
+
+#include "metadata_suite.h"
+#include "unifycr_meta.h"
+#include "unifycr_metadata.h"
+#include "t/lib/tap.h"
+#include "t/lib/testutil.h"
```

Comment:
Same as before with `t/lib/testutil.h`. Looks like you're not using anything from there.